packages/components/src/dropdown-menu-v2/index.tsx

```diff
@@ -108,9 +108,11 @@ const UnconnectedDropdownMenu = (
 	);
 
 	// Extract the side from the applied placement — useful for animations.
+	// Using `currentPlacement` instead of `placement` to make sure that we
+	// use the final computed placement (including "flips" etc).
 	const appliedPlacementSide = useStoreState(
 		dropdownMenuStore,
-		'placement'
+		'currentPlacement'
```

Comment:
I was surprised this is an undocumented menu store feature (seems to be a popover store feature actually). Should we raise an issue so it gets documented? 

I assume you borrowed this from the [Framer Motion example](https://ariakit.org/examples/menu-framer-motion), but still, it might be difficult to find at a first glance.

Reply:
I had already [opened an issue for it](https://github.com/ariakit/ariakit/issues/4082) — and now I replied asking for this to be documented.